tests/device_tests/ethereum/test_sign_eip7702.py

```
@@ -0,0 +1,91 @@
+
+    def _sign() -> EthereumAuth7702Signature:
+        return ethereum.sign_auth_eip7702(
+            session,
+            n=parse_path(parameters["path"]),
+            chain_id=parameters["chain_id"],
+            delegate=delegate,
+            nonce=parameters["nonce"],
+            encoded_network=defs.encoded_network,
+        )
+
+    with pytest.raises(TrezorFailure, match="Experimental features are disabled."):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)